cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -360,7 +360,7 @@ static void sc_mkdir_and_mount_and_glob_files(const char *rootfs_dir,
 static void sc_mount_nvidia_driver_biarch(const char *rootfs_dir, const char **globs, size_t globs_len)
 {
 
-	const char *native_sources[] = {
+	static const char *native_sources[] = {
```

Comment:
Slightly changes the meaning now. I guess the effect is obtained if this is moved outside just as `const`, but it also would have to be declared with static (again slightly different meaning) as it's local to this *.o. I guess it's better if the arrays are kept close to the place where they are used.

Reply:
Yes, this preserves the existing visibility of the variable, it just changes its storage to be static. It's unfortunate that in C/C++ the `static` keyword as a handful of totally different meanings :-)